spaces/S000046/properties/P000056.md

```diff
@@ -7,5 +7,11 @@ refs:
   name: Counterexamples in Topology
 ---
 
+Each interval $U_n=(0,1/n)$ for $n\ge 2$ is open in $X$ and dense since $X$ is {P39}.
+So its complement $X\setminus U_n$ is nowhere dense.
+The intersection of all the $U_n$ is empty.
+Therefore, the union of all the complements is the whole space.
+Thus $X$ is a countable union of nowhere dense sets.
```

Comment:
```suggestion
Each interval $U_n=(0,1/n)$ for $n\ge 2$ is open in $X$ and dense since $X$ is {P39}.
So its complement $U_n^c=X\setminus (0,1/n)$ is nowhere dense, and
thus $X$ is a countable union of nowhere dense sets $U_n^c$.
```